client/js/components/map/admin/AdminLayerListItem.vue

```diff
@@ -321,7 +321,7 @@ export default {
        *
        */
       const toggleMyIconInSameGroup = (this.isLinkedWithCurrentlyHovered && this.showCurrentIconState)
-      const toggleMyIconWithoutGroup = (this.showCurrentIconState && this.visibilityGroupIdOfHoveredLayer === '')
+      const toggleMyIconWithoutGroup = (this.showCurrentIconState && (this.visibilityGroupIdOfHoveredLayer === '' || this.visibilityGroupIdOfHoveredLayer !== null))
```

Comment:
Not sure if I fully understand this, but from the PR description I would expect this to be
```
(this.visibilityGroupIdOfHoveredLayer === '' || this.visibilityGroupIdOfHoveredLayer === null))
```
i.e. `visibilityGroupIdOfHoveredLayer` has the default value or is not set.

If we really should check for `!== null`, this would include `=== ''`, so we could remove the first check.